packages/panna-sdk/src/react/components/account/account-dialog.tsx

```diff
@@ -28,35 +28,42 @@ import {
 import type { DialogStepperContextValue } from '../ui/dialog-stepper';
 import { Separator } from '../ui/separator';
 import { Tabs, TabsContent, TabsList, TabsTrigger } from '../ui/tabs';
+import { AccountScreensProvider } from './account-screens-provider';
 import { AccountSettingsView } from './account-settings-view';
-
-enum AccountViewEnum {
-  Main = 'main',
-  Settings = 'settings',
-  Buy = 'buy',
-  Send = 'send'
-}
-
-type AccountView = `${StringValues<typeof AccountViewEnum>}`;
+import {
+  AccountView,
+  AccountViewEnum,
+  useAccountView
+} from './account-view-provider';
 
 type AccountDialogProps = {
   address: string;
 };
 
 export function AccountDialog({ address }: AccountDialogProps) {
-  const [activeView, setActiveView] = useState<AccountView>(
-    AccountViewEnum.Main
-  );
+  const { activeView, setActiveView } = useAccountView();
+  const [activeTab, setActiveTab] = useState<string>('balance');
   const buyStepperRef = useRef<DialogStepperContextValue | null>(null);
   const [sendStepperContext, setSendStepperContext] =
     useState<DialogStepperContextValue | null>(null);
+  const [sendCollectibleStepperContext, setSendCollectibleStepperContext] =
+    useState<DialogStepperContextValue | null>(null);
 
   const { data: balanceUsd = 0, isLoading: isLoadingUsdBalance } =
     useTotalFiatBalance({
       address,
       currency: DEFAULT_CURRENCY
     });
 
+  const handleTabClick = (tab: string) => {
+    setActiveTab(tab);
+  };
+
+  const handleCollectibleClose = () => {
+    setActiveView(AccountViewEnum.Main);
+    setActiveTab('collectibles');
```

Comment:
Can we please define constants (or enum entries in AccountViewEnum, as applicable) for `collectibles` and `balance` on L45 above and reuse them across the code to avoid any typos in the future?